build.sbt

```diff
@@ -0,0 +1,41 @@
+lazy val supportedScalaVersions = List("2.13.4", "2.12.12")
+
+ThisBuild / organization := "org.tree_annotation"
+ThisBuild / scalaVersion := "2.12.12"
+ThisBuild / name := "tao-parser"
+ThisBuild / description := "first truly universal syntax for structural communication at any scale, born of a vision of total intercommunication of all software systems"
+ThisBuild / licenses := List("MIT" -> url("https://choosealicense.com/licenses/mit/"))
+ThisBuild / homepage := Some(url("https://www.tree-annotation.org/"))
+ThisBuild / scmInfo := Some(ScmInfo(url("https://github.com/tree-annotation/tao"), "https://github.com/tree-annotation/tao.git"))
+
+ThisBuild / developers := List(Developer(
+  "Truman A. Oz",
+  "Truman A. Oz",
+  "truman\" + \"!@#\".charAt(1) + \"tree-annotation.org",

```

Comment:
the inner quotes should be unescaped

Reply:
this is sbt escaping

Reply:
I changed it to raw string, check if this is like you want now.

Reply:
The idea is to confuse possible spambots looking at this source code.

Reply:
ahh, I completely missed the point. I thought the goal is to make this code to the final output so only programmers can read it or something like that.